### PR TITLE
19 inventory

### PR DESCRIPTION
**Pull request by vbroda17**
Added Inventory React Component to satisfy the business view I believe. 

This includes layout, css, and linking functionality with the back-end endpoints that we currently have.

### CODE REVIEW

File: src/pages/Inventory.tsx [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+  const handleInputAboutChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setItemAbout(event.target.value);
+  };
+
+  const handleAddItem = async () => {
+    if (itemName.trim() !== "") {
+      try {
+        const newItem = {
+          name: itemName,
+          id: generateRandomId(),
```
Comment (noahshoap) [inline src/pages/Inventory.tsx:32]: Do we want to remove the ID thing for adding items?  That should be handled by the backend I think -- I believe the database auto-increments the item ID.

File: src/pages/Inventory.tsx [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+        </button>
+      </div>
+      <div id="sales">
+        <h3>Sales?</h3>
+      </div>
+    </div>
+  );
+}
+
+interface Item {
```
Comment (noahshoap) [inline src/pages/Inventory.tsx:207]: Just an idea, but what do you think about making a new folder to store this & other interfaces/classes?

Something like `Abstractions` for interfaces, `Models` for classes.

Then importing it in the relevant code when needed.

File: src/pages/Inventory.tsx [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+  const handleSaveItem = async () => {
+    if (selectedItem) {
+      try {
+        const updatedItem = {
+          id: selectedItem.id,
+          price: itemPrice,
+          quantity: itemQuantity,
+        };
+
+        const response = await fetch(`https://localhost:7079/api/inventory/update/`, {
```
Comment (noahshoap) [inline src/pages/Inventory.tsx:82]: I think we should consider changing this to be a const variable somewhere.

As in,
`const inventoryApiPath = "https://localhost:7079/api/inventory";`

then using it in code like
`const response = await fetch('${inventoryApiPath}/update');`

**Review by noahshoap — APPROVED** [review]
Overall, looks good to me!  I did comment a few thoughts I had, but feel free to ignore those if you don't want to do them.

**Comment by vbroda17** [comment]
I think all of Noah's points makes sense. If all right I might make issues for them or make the changes the next time I am working with the front end but they should be simple changes.

**Pull request merged**